src/lib.rs

```diff
@@ -237,7 +250,20 @@ mod tests {
     }
 
     #[test]
-    fn test_fasttest_new_default() {
+    fn test_fasttext_new_default() {
         let _fasttext = FastText::default();
     }
+
+    #[test]
+    fn test_fasttext_get_word_vector() {
+        let mut _fasttext = FastText::default();
```

Comment:
You should remove the `_` prefix because the variable is used.

Reply:
`_` prefix usually denotes intended unused variable to avoid the compiler warning us.

Reply:
Fixed.